### PR TITLE
Resolve bugs, add and update features

### PR DESCRIPTION
**Pull request by jeannetoh99**
Resolve the following bugs:
1. Check for duplicates in edit, move and copy commands
2. Handle Duplicate Error in Json file on initialising app
3. Removed List Command

Add the following features:
1. Auto Tab after add/edit/delete command

Update the following features:
1. Change isSameFriend to check for passport number only #222
2. Include sort -friend passport and mobile #202
3. Sort activity by most importance ones first instead of least important #224
4. Change isSameActivity to check for name, date and location #235

### CODE REVIEW

File: src/main/java/seedu/address/model/travelplan/exceptions/DuplicateTravelPlanException.java [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+
 /**
  * Signals that the operation will result in duplicate TravelPlans (TravelPlans are considered duplicates if they have
  * the same
  * identity).
  */
-public class DuplicateTravelPlanException extends RuntimeException {
+public class DuplicateTravelPlanException extends DuplicateException {
     public DuplicateTravelPlanException() {
         super("Operation would result in duplicate persons");
```
Comment (underthehai) [inline src/main/java/seedu/address/model/travelplan/exceptions/DuplicateTravelPlanException.java:12]: duplicate friends?

**Review by underthehai — APPROVED** [review]
LGTM, just the minor naming part

File: src/main/java/seedu/address/model/travelplan/exceptions/DuplicateTravelPlanException.java (hunk 1 above)
Comment (jeannetoh99) [reply src/main/java/seedu/address/model/travelplan/exceptions/DuplicateTravelPlanException.java]: updated to duplicate travel plans! :) thanks!

**Review by jiaweiteo — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/friend/Friend.java [hunk 2]
```diff
@@ -51,15 +52,15 @@ public Mobile getMobile() {
     }
 
     /**
-     * Returns true if both friends are of the same name
+     * Returns true if both friends have the same passport or same mobile.
```
Comment (lyeyixian) [inline src/main/java/seedu/address/model/friend/Friend.java:55]: This doesn't include mobile isn't it?

**Review by lyeyixian — APPROVED** [review]
LGTM. Just one JavaDoc typo I think.

File: src/main/java/seedu/address/model/friend/Friend.java (hunk 2 above)
Comment (jeannetoh99) [reply src/main/java/seedu/address/model/friend/Friend.java:55]: right!

**Pull request merged**